src/ngraph/runtime/gpu/op/lstm.hpp

```diff
@@ -0,0 +1,103 @@
+/*******************************************************************************
+* Copyright 2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include "ngraph/op/util/requires_tensor_view_args.hpp"
+#include "ngraph/util.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace gpu
+        {
+            class Lstm : public util::RequiresTensorViewArgs
+            {
+            public:
+                // INPUTS:
+                // [0] - xt, input tensor of layout TNC, Shape{sequence length*batch_size, feature_size}
```

Comment:
Use doxygen comments like `/// \brief`, `/// \param`
If you use the right syntax then the compiler will complain if you do something wrong or forget to document a parameter